tmd/io/io.py

```diff
@@ -162,16 +162,25 @@ def load_population(neurons, user_tree_types=None, name=None, use_morphio=False)
     elif os.path.isdir(neurons):  # Assumes given input is a directory
         files = [os.path.join(neurons, neuron_dir) for neuron_dir in os.listdir(neurons)]
         name = name if name is not None else os.path.basename(neurons)
+    elif os.path.isfile(neurons):  # Assumes given input is a file
+        files = [neurons]
+        name = name if name is not None else os.path.basename(neurons)
+    else:
+        raise TypeError(
+            f'The type of the given neurons ({type(neurons)}) is not supported or the path does '
+            'not exist',
+        )
 
     pop = Population.Population(name=name)
 
     for filename in files:
         try:
+            ext = os.path.splitext(filename)[-1][1:]
```

Comment:
Isn't the `ext = Path(filename).suffix` a bit safer? Also for readability would leave the dots in, e.g. ".h5" to avoid the slicing.

Reply:
In `load_neuron` we use `os.path.splitext` so I used the same for consistency. But I can change if you prefer.
And for the slicing it was just to not always compare the starting `.` but it's no big deal, I can revert it.

Reply:
Nah, that's fine.